src/topology/algebra/ordered.lean

```diff
@@ -9,17 +9,25 @@ import data.set.intervals
 import topology.algebra.group
 import topology.constructions
 
-/-! # Theory of ordered topology
+/-! # Theory of topology on ordered spaces
 
 ## Main definitions
-`ordered_topology` and `orderable_topology`
 
-TODO expand
+The order topology on an ordered space is the topology generated by all open intervals (or
```

Comment:
I think giving its Lean name would help clearing the confusion with the Prop-valued classes discussed below.

Reply:
Done